tsconfig.typedoc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "lib": ["ESNext"],
+    "module": "esnext",
+    "target": "esnext",
+    "moduleResolution": "node",
+    "skipLibCheck": true,
+
+    // so that if your project isn't using TypeScript, it still has autocomplete
```

Comment:
is this syntax allowed in JSON?

Reply:
That's copied from directly from `tsconfig.json`,  I believe so! More than happy to remove the comments from both files though as they're relics of the Bun new project `tsconfig.json` template.

Reply:
It's invalid JSON so if it's working they're probably using something like `json5` for parsing.

Reply:
Yep, that seems to be the case per some googling on the matter (updated the original comment w link to the Typescript PR adding support for comments in `tsconfig.json` file) ^^

Reply:
cool, will land!